index.js

```diff
@@ -32,19 +38,22 @@ function read (input) {
 }
 
 function parse (input) {
-  let parsed
+  if (jsonIsArray.test(input)) {
+    return parseJson(input)
+  }
+
+  return input
+    .split('\n')
+    .filter(l => !isEmpty.test(l))
```

Comment:
Can we use a more descriptive parameter name here, `line => !isEmpty.test(line)`